### DISCUSSION TITLE
Where can I find which youtube-dl version is currently being used by DLG?

### DISCUSSION THREAD

**Discussion by One-Hoopy-Frood** [oleksis/youtube-dl-gui#120] [Q&A]
I'd like to be sure I'm using the newest version of youtube-dl that DLG is using and WHERE it's located. I've used the 'update' function, but it doesn't give useful output like which version of youtube-dl is being used and where that is being stored (path).

Could we get that info on the "about" page?

**Comment by oleksis** [comment]
Please read issue #121
